fashion_mnist/README.md

```diff
@@ -0,0 +1,394 @@
+# Fashion MNIST Tutorial
+
+In this tutorial, you will learn how to apply Horovod to a [WideResNet](https://arxiv.org/abs/1605.07146) model, trained on Fashion MNIST dataset.
+
+If this is an in-person session, hosts will set up VM for you and provide you credentials to Jupyter Lab.  If you're working on this tutorial on your own, please follow installation instructions in [INSTALL.md](INSTALL.md).
+
+Let's begin!
+
+## Connect to Jupyter Lab
```

Comment:
How do you connect to Jupyter lab? If it's run or click X, recommend specifying that here.